anthropic/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
```

Comment:
@dpoint01 Do you have any list of metrics that you would want added here ?

Reply:
@barieom Yes but It's probably not a good idea since they need to be duplicated across all integrations. They also depend on what we trace in the integration itself.
The idea we should pursue - later - is to have an LLM Obs page in the public docs where we list all metrics we collect and link it here.

So for now I would say we keep this empty.